firefox-ios/Client/Frontend/Browser/Tabs/LayoutManager/TabsSectionManager.swift

```diff
@@ -26,7 +26,13 @@ class TabsSectionManager {
 
     func layoutSection(_ layoutEnvironment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection {
         let availableWidth = layoutEnvironment.container.effectiveContentSize.width
-        let maxNumberOfCellPerRow = Int(availableWidth / UX.cellEstimatedWidth)
+        let maxNumberOfCellsPerRow = Int(availableWidth / UX.cellEstimatedWidth)
+        let minNumberOfCellsPerRow = 2
+
+        var numberOfCellsPerRow = maxNumberOfCellsPerRow
+        if numberOfCellsPerRow < minNumberOfCellsPerRow {
+            numberOfCellsPerRow = minNumberOfCellsPerRow
+        }
```

Comment:
```suggestion
        let numberOfCellsPerRow = maxNumberOfCellsPerRow < minNumberOfCellsPerRow ? minNumberOfCellsPerRow : maxNumberOfCellsPerRow
```
Small nit (and a matter of personal preference) I think this could be done with a ternary operation.
Also I think it will be good to write a small comment to understand this change quickly